build_runtimes.sh

```diff
@@ -0,0 +1,3 @@
+cargo b -r --features with-base-runtime
+cp target/release/trappist-collator bin/base-collator
```

Comment:
```suggestion
mc target/release/trappist-collator target/release/base-collator
```

instead of copying to bin, i would just rename it in the same target/release folder so we have both collators under the same folder (kind of expected)

Reply:
Agree